### ISSUE TITLE
Create Model

### ISSUE THREAD

**Issue by QColeman97** [QColeman97/AudioTagger#4]
2D CNN's promising via Kaggle.
1 on leaderboard: http://dcase.community/documents/challenge2018/technical_reports/DCASE2018_Jeong_102.pdf
2 on leaderboard: https://cpjku.github.io/dcase_task2/
3 on leaderboard: http://dcase.community/documents/challenge2018/technical_reports/DCASE2018_Iqbal_89.pdf

Ensemble of top methods?
https://www.kaggle.com/daisukelab/freesound-dataset-kaggle-2018-solution

ch 7: (advanced prac) 
      model ensembling (of 2D CNN & Combined 1D CNN & RNN combo, etc??)
   Replace Conv2D w/ DepthwiseConv2D (or maybe SeperableConv2D?)
   Batch normailization
   Inception
   Residual connections
   DenseNet?

 ch 6: (Sequential methods)
    Staley - 1D CNN is preprocessing step before RNN
 .  Bidirectional RNNs, recurrent dropout, & stacking RNNS

 ch 5: (Convolution)
   Tune HP (# neurons, layers, epochs, batch_size)
 .  Dropout, regularization
 .  Data augmentation - No
 .  Use a pretrained CNN? (VGG16)
 .  Fine-tuning (VGG16)

K-Fold Validation
